modules/backup-gcs/client.go

```diff
@@ -216,32 +182,32 @@ func (g *gcsClient) PutObject(ctx context.Context, backupID, key string, byes []
 	return nil
 }
 
-func (g *gcsClient) Initialize(ctx context.Context, backupID string) error {
+func (g *gcsClient) Initialize(ctx context.Context, backupID, bucketName, bucketPath string) error {
 	key := "access-check"
 
-	if err := g.PutObject(ctx, backupID, key, []byte("")); err != nil {
-		return errors.Wrap(err, "failed to access-check gcs backup module")
+	if err := g.PutObject(ctx, backupID, key, bucketName, bucketPath, []byte("")); err != nil {
+		return errors.Wrap(err, "failed to access-check gcs backup module"+bucketName+bucketPath+backupID+key)
 	}
 
-	bucket, err := g.findBucket(ctx)
+	bucket, err := g.findBucket(ctx, "")
```

Comment:
this second param should be an empty string? or here we should pass `bucketName`?